TYPO3.Fluid/Classes/TYPO3/Fluid/Core/Parser/SyntaxTree/ObjectAccessorNode.php

```diff
@@ -96,7 +96,9 @@ public static function getPropertyPath($subject, $propertyPath, RenderingContext
                 if (is_object($subject) && preg_match('/^(is|has)([A-Z].*)/', $propertyName, $matches) > 0) {
                     try {
                         $subject = $subject->$propertyName();
-                    } catch (\Exception $exception) {
+                    } catch (\Throwable $exception) {
+                        $subject = null;
+                    } catch (\Error $exception) {
```

Comment:
`\Throwable` would catch `\Error` since it extends it

and this wouldn't work in PHP 5.5/5.6 since `\Exception` would be thrown there..

so it has to be

    } catch (\Throwable $exception) {
        $subject = null;
    } catch (\Exception $exception) {
         $subject = null;
    }